Cakefile

```diff
@@ -54,6 +54,10 @@ codeFor = ->
     js = js.replace /^\/\/ generated.*?\n/i, ''
 
     cshtml = "<pre><code>#{hljs.highlight('coffeescript', cs).value}</code></pre>"
+    # Temporary fix until highlight.js adds support for newer CoffeeScript reserved words
```

Comment:
I think it's fine special-case here. However, I think we should use the power of regex instead of a `for` loop ;)

```coffee
cshtml = cshtml.replace /import|export|from|as|default/g, '<span class="reserved">$&</span> '
```